### PR TITLE
Tg mapconflict helper

### PR DESCRIPTION
**Pull request by Crazylemon64**
"Borrowed" from tgstation/tgstation#22421. Credits to @xxalpha for the original PR!

With any luck, this should allow us to release the deadlock on the map held by massive PRs, allowing for multiple people editing the map at the same time to not have to resort to torching their work and starting from the top, manifold times.

### CODE REVIEW

**Comment by Crazylemon64** [comment]
Alright, now to test it on a couple PRs (on my own end, of course) to see if it'll do what I want it to

**Comment by Crazylemon64** [comment]
Tested on #6076 + #6111, with the following results:
conflictfixer_liftoffa
conflictfixer_liftoffb

A rousing success! Map tokens are dead!

File: code/modules/admin/verbs/mapping.dm [hunk 1]
```diff
@@ -30,6 +30,12 @@ var/intercom_range_display_status = 0
 		src.pixel_x = -224
 		src.pixel_y = -224
 
+/obj/effect/debugging/mapfix_marker
```
Comment (Fox-McCloud) [inline code/modules/admin/verbs/mapping.dm]: What does this do, precisely?

Comment (Crazylemon64) [inline code/modules/admin/verbs/mapping.dm]: It shows up in the new map if they modify the same tile, and is evidence that the map was not correctly de-conflicted.

Comment (xxalpha) [inline code/modules/admin/verbs/mapping.dm]: It's to aid the mapper in fixing the conflicts, he won't have to hunt down for messed up tiles.

**Comment by AndriiYukhymchak** [comment]
111 rows of .py awesomness

File: tools/map_conflict_fixer/map_helpers.py [hunk 2]
```diff
@@ -639,8 +639,12 @@ def fix_map_git_conflicts(base_map, ours_map, theirs_map, mode, marker, priority
                     grid_conflict_counter += 1
 
     merged_dict = sort_dictionary(merged_dict)
-    write_dictionary_tgm(path+".fixed.dmm", merged_dict)
-    write_grid_coord_small(path+".fixed.dmm", merged_grid, maxx, maxy)
+	if(tgm)
+	    write_dictionary_tgm(path+".fixed.dmm", merged_dict)
+	    write_grid_coord_small(path+".fixed.dmm", merged_grid, maxx, maxy)
+	else
+        write_dictionary(path+".fixed.dmm", old_dict)
```
Comment (tigercat2000) [inline tools/map_conflict_fixer/map_helpers.py]: is this supposed to be old_dict

Comment (Crazylemon64) [inline tools/map_conflict_fixer/map_helpers.py]: Yeah, I made a commit at the end that fixed that to be `merged_dict`

**Pull request merged**